doc/nrf/libraries/security/tfm/tfm_ioctl_api.rst

```diff
@@ -29,6 +29,32 @@ The supported platform services are defined by :c:struct:`tfm_platform_ioctl_cor
 
 Set the :kconfig:option:`CONFIG_TFM_PARTITION_PLATFORM` Kconfig option to enable the services.
 
+Read service
+============
+
+The TF-M IOTCL read service allows the NSPE to read memory areas from the SPE when they are not accessible from the NSPE.
```

Comment:
Nit,  so feel free to ignore the suggestion if it is a hassle to do but the last part of the sentence it feels a bit off to me. 
Maybe something like this would be a bit more clear:

to access memory areas within the SPE that would otherwise be inaccessible to it.

Otherwise it looks good.

Reply:
I'm not opposed to this wording.

Reply:
I'll update.